library/src/main/java/com/leinardi/android/speeddial/UiUtils.java

```diff
@@ -98,6 +107,9 @@ public void run() {
                     }
                 })
                 .start();
+        // uncomment to show that NullAway is actually running
+                 Object x = null;
+                 x.hashCode();
```

Comment:
**Reporter**: Error Prone
**Rule**: NullAway
**Severity**: ERROR
**File**: library/src/main/java/com/leinardi/android/speeddial/UiUtils.java L112

dereferenced expression x is @Nullable

x.hashCode();^(see http://t.uber.com/nullaway )

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<1456516635>*